From c5d8fc0004a2c76244263edc01adb9fa05f26a77 Mon Sep 17 00:00:00 2001
From: Jeff Horn <jhorn@reachtech.com>
Date: Fri, 8 Nov 2013 09:52:08 -0800
Subject: [PATCH] Remove old Xf86 input call.

The function is not available in the current version
of X11

Signed-off-by: Jeff Horn <jhorn@reachtech.com>
---
 src/tslib.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/tslib.c b/src/tslib.c
index b6976ab..d466681 100644
--- a/src/tslib.c
+++ b/src/tslib.c
@@ -194,10 +194,6 @@ static void ReadInput (InputInfoPtr local)
 			x = samp.x;
 			y = samp.y;
 
-			xf86XInputSetScreen(local, priv->screen_num,
-					samp.x,
-					samp.y);
-
 			xf86PostMotionEvent (local->dev, TRUE, 0, 2,
 					x, y);
 
-- 
1.8.1.2

